index.html

```diff
@@ -4,53 +4,71 @@
 <head>
   <meta name="viewport" content="width=device-width">
   <meta name="description" content="James H. Nguyen's Front End Developer Portfolio">
-  <link rel="stylesheet" href="main.css"/>
+  <link rel="stylesheet" href="css/index.css"/>
   <style>
-    @import url(https://fonts.googleapis.com/css?family=Anonymous+Pro);
-    @import url('https://fonts.googleapis.com/css2?family=Source+Code+Pro:wght@600&display=swap');
+    @import url('https://fonts.googleapis.com/css2?family=Share+Tech&display=swap');
+    @import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;800&display=swap');
+
+    @media only screen and (max-wdith: 942px) {
+  body {
+    background-color: green;
+  }
+}
+
   </style>
   <title>James H. Nguyen's Portfolio</title>
 </head>
 
 <body>
-
-  <section>
-    <h1>James H. Nguyen</h1>
-    <div class="typeWrapper">
-      <div class="typewriter">
-        <p>Front End Developer</p>
+  <!-- <section class = "header" h1-hover-after="Front End Developer"> -->
+    <!-- James H. Nguyen -->
+    <div class = "nav-bar">
```

Comment:
```suggestion
    <div class="nav-bar">
```
Just a code formatting this: There should be an option in your IDE that can clean it up for you.